src/authn/Authenticator.ts

```diff
@@ -18,7 +18,7 @@ export default class Authenticator {
   async createToken(timestamp?: Date): Promise<Token> {
     const authData = AuthData.create(
       this.identityKey.publicKey.walletSignatureAddress(),
-      timestamp || new Date()
+      timestamp || new Date(Date.now() - 5000)
```

Comment:
Hm, this seems like an odd thing to do. Is this about clocks being out of sync between client and server? Would it make more sense to expose a setting on the server side to allow tweaking the clock skew tolerance?

Reply:
Yep I think that's what's happening there, docker clock out of sync. I've actually only seen it happen locally too, not in CI, but a bunch of times. 

Do you mean pass the skew in as a server option here instead of hard-coding it?